Makefile

```diff
@@ -1,7 +1,7 @@
 RHEL5=$(shell grep -q -i 'release 5' /etc/redhat-release 2>/dev/null || false)
 PYTHON3=$(shell which python3 && which pip3)
 USE_SELINUX=$(shell test -d /sys/fs/selinux && echo ":Z")
-DOCKERFILE  ?= $(CURDIR)/images/Dockerfile.el7
```

Comment:
Why is it needed to change the default?

Reply:
The "which" import in line https://github.com/Katello/katello-host-tools/blob/de6121ea1935f531c903a71695feb67fcc518828/src/katello/deb_tracer.py#L3 is not known by the python version on the el7 container

Reply:
Then that's a problem that you should address. In CI all the docker files run. el7 is just the default here. In the unittest_suite there's code to separate out Python 2 and Python 3.